frontends/infinite-corridor/src/api/fields/hooks.ts

```diff
@@ -10,11 +10,11 @@ import axios from "../../libs/axios"
 import { fieldDetails } from "./urls"
 
 const useFieldsList = () => {
-  return useQuery<FieldList>(urls.fieldsList)
+  return useQuery<FieldList>([urls.fieldsList])
```

Comment:
In React Query v4, query keys **must** be arrays. Previously they could be arrays or strings.